monai/transforms/spatial/array.py

```diff
@@ -346,7 +346,7 @@ def update_meta(self, img: torch.Tensor, dst_affine=None, img_dst=None):
         if dst_affine is not None:
             super().update_meta(img, dst_affine)
         if isinstance(img_dst, MetaTensor) and isinstance(img, MetaTensor):
-            original_fname = img.meta[Key.FILENAME_OR_OBJ]
+            original_fname = img.meta.get(Key.FILENAME_OR_OBJ, "resample_to_match_source")
```

Comment:
May I know what this "resample_to_match_source" filename means? Maybe "None", "not provided" or some macro definition?

Thanks.

Reply:
Agreed, "not provided" makes more sense.

Reply:
Agreed, "not provided" makes more sense.

Reply:
That's mainly to make sure we know that the value is editted by this transform. If there are multiple transforms can set this value in the futur, using a common name will make it difficult to find out which transform initially added this key-value